src/main/java/lesser/brickbreaker/BrickBreakerFrame.java

```diff
@@ -0,0 +1,95 @@
+package lesser.brickbreaker;
+
+import levy.brickbreaker.Ball;
+import levy.brickbreaker.Paddle;
+import levy.brickbreaker.Brick;
+import reiff.brickbreaker.Controller;
+
+import javax.swing.*;
+import java.awt.event.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BrickBreakerFrame extends JFrame {
+    private static final int COLS = 10;
+    private static final int ROWS = 5;
+    private static final int BRICK_WIDTH = 60;
+    private static final int BRICK_HEIGHT = 20;
+    private static final int SPACING = 10;
+
+    private final Ball ball = new Ball(390, 510, 20, 20, 20, 5, 45);
+
+    private final Paddle paddle = new Paddle(350, 550, 100, 10, 60);
+    private final List<Brick> bricks = new ArrayList<>();
+
+    private final BrickBreakerComponent view = new BrickBreakerComponent(ball, paddle, bricks);
+    private boolean ballMoving = false;
+    private final Controller controller = new Controller(ball, paddle, bricks, view);
+
+    public BrickBreakerFrame() {
+        setSize(800, 600);
+        setTitle("Brick Breaker");
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+        setLocationRelativeTo(null);
+
+        add(view);
+        view.setBounds(0, 0, 800, 600);
+
+        setFocusable(true);
+        requestFocusInWindow();
+
+        initializeBricks();
+
+        addKeyListener(new KeyAdapter() {
+            @Override
+            public void keyPressed(KeyEvent e) {
+                int keyCode = e.getKeyCode();
+                int paddleSpeed = paddle.getSpeed();
+
+                // Move paddle left or right
+                if (keyCode == KeyEvent.VK_LEFT) {
+                    paddle.x = Math.max(0, paddle.x - paddleSpeed);
+                } else if (keyCode == KeyEvent.VK_RIGHT) {
+                    paddle.x = Math.min(view.getWidth() - paddle.width, paddle.x + paddleSpeed);
+                }
+
+                if (keyCode == KeyEvent.VK_UP && !ballMoving) {
+                    ballMoving = true;
+                    controller.startGame();
+                }
+
+                view.repaint();
```

Comment:
This all belongs in the controller

Reply:
Thank you! I modify this to be controlled by the controller now and based on that I changed the way the paddles are moved in networksPlay() to be managed with keyboard, however it still won't run